src/app/screens/Liquid/ConfirmGetPublicKey.tsx

```diff
@@ -0,0 +1,116 @@
+import { CheckIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import PublisherCard from "@components/PublisherCard";
+import Checkbox from "@components/form/Checkbox";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+import ScreenHeader from "~/app/components/ScreenHeader";
+import { useNavigationState } from "~/app/hooks/useNavigationState";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import msg from "~/common/lib/msg";
+import { OriginData } from "~/types";
+
+function LiquidConfirmGetPublicKey() {
+  const { t } = useTranslation("translation", {
+    keyPrefix: "nostr",
```

Comment:
this should be liquid here - this template should not share the prefix with anything else